app/react-graphql/components/shared/MutatorButton.tsx

```diff
@@ -8,5 +8,5 @@ export interface MutatorButtonProps {
 
 export default function MutatorButton(props: MutatorButtonProps & ButtonProps) {
   const {state, ...rest} = props;
-  return <Button {...rest} onPress={state.executeMutation} />;
+  return <Button {...rest} onPress={()=> state.executeMutation()} />;
```

Comment:
onPress has event args that were messing up state.executeMutation by tricking it into applying the event args as variables.